src/test/java/attendance/domain/AttendanceBookTest.java

```diff
@@ -0,0 +1,79 @@
+package attendance.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import attendance.TestUtil;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class AttendanceBookTest {
+
+    @DisplayName("해당 크루의 출석 기록을 추가하고 조회할 수 있다.")
+    @Test
+    void test_addCrewRecord() {
+        // given
+        Map<String, AttendanceHistory> map = new HashMap<>();
+        String crewName = "빙티";
+        map.put(crewName, new AttendanceHistory());
+        AttendanceBook attendanceBook = new AttendanceBook(map);
+        AttendanceRecord record = TestUtil.createRecord(LocalDateTime.of(2024, 12, 2, 13, 0));
+
+        // when
+        attendanceBook.add(crewName, record);
+
+        // then
+        AttendanceHistory history = attendanceBook.getHistoryByName(crewName);
+        assertThat(history.getRecords()).hasSize(1);
+        assertThat(history.getRecords().contains(record)).isTrue();
```

Comment:
```suggestion
        assertThat(history.getRecords()).contains(record);
```

assertj 에서 제공하는 `contains()` 를 쓰면 더 간결하게 표현할 수 있습니다.

Reply:
그렇군요, 수정했습니다..! 배워갑니다.